doc/manual/manual.tex

```diff
@@ -2286,6 +2286,58 @@ \subsection{Advection Stabilization}
 \end{itemize}
 
 
+Both add additional terms to the temperature (or compositional field equation). We will discuss
+the case for the temperature equation here. The compositional fields only differ in having
+a zero conductivity, fewer right-hand side terms, and $\rho C_p=1$. The strong form of the temperature equation reads
+\[
+ \rho C_p \frac{\partial T}{\partial t} + \rho C_p \mathbf{u} \cdot \nabla T - \nabla \cdot k\nabla T = F,
+\]
+where $F$ is the combination of source and reaction terms,
+while the weak form (with test function $\varphi$) becomes:
+\[
+a(T,\varphi) =
+ \left(\rho C_p \frac{\partial T}{\partial t}, \varphi \right)
+ + \left(\rho C_p \mathbf{u} \cdot \nabla T, \varphi \right)
+ + \left( k \nabla T, \nabla \varphi \right) = (F,\varphi) = f(\varphi).
+\]
+
+\subsubsection{SUPG Stabilization}
+
+For streamline upwind/Petrov-Galerkin (SUPG) (see for example \cite{JohnKnobloch2006,step-63}), we amend the weak form with a cell-wise
+defined weak form
+\[
+a_{\text{SUPG}} (T, \varphi) =
+ \sum_{K \in \mathcal{T}_h}
+  \delta_K \left( \rho C_p \frac{\partial T}{\partial t} - k \triangle T + \mathbf{\beta} \cdot \nabla T - F, \mathbf{\beta} \cdot \nabla \varphi \right)_K,
+\]
+where $K \in \mathcal{T}_h$ are the cells in the computation, $\delta_K \geq 0$ is a stabilization coefficient
+defined on each cell, $\mathbf{\beta} = \rho C_p \mathbf{u}$ is the effective advection velocity.
```

Comment:
Is there a reason you replace this by beta?

Reply:
Because I don't want to write this thing that many times and it helps to know that this is, effectively, the velocity we are advecting with (the normal SUPG literature doesn't have rho C_p, just a velocity. I made the mistake the first time not to include the constants and that makes the method terrible. I am adding a sentence.